### PR TITLE
WIP: Marshmallow defs

### PR DESCRIPTION
**Pull request by cmungall**
cc @dougli1sqrd

### CODE REVIEW

File: ontobio/assoc_schema.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from marshmallow import Schema, fields, pprint
+
```
Comment (dougli1sqrd) [inline ontobio/assoc_schema.py]: So then do you use these to produce classes? Are these like "meta" classes? Or do you instantiate these directly?

**Comment by cmungall** [comment]
marshmallow.readthedocs.io/en/latest/

To instantiate a class:
https://marshmallow.readthedocs.io/en/latest/quickstart.html#deserializing-objects-loading

codegen for the class: ??

@kshefchek of relevance to phenopackets

**Comment by cmungall** [comment]
Somewhat indirect: https://github.com/cwacek/python-jsonschema-objects

**Comment by dougli1sqrd** [comment]
And then you'd have a User (from their example) class written somewhere that knows how to be instantiated with a **dict object, and then the @post_load decorator will actually instantiate the given class (ie User)?

**Pull request merged**